index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *Gregory-Boyd card START ________  -->
+    <div class="card">
+      <p class="name">Gregory Boyd</p>
+      <p class="contact">
+        <i class="fab fa-linkedin"></i>
+        <a href="https://www.linkedin.com/in/gregory-boyd-429a56207/ target="_blank">Gregory Boyd Linkedin Profile</a>
```

Comment:
You have a syntax error in this line.
The hyperlink should be closed with a " symbol.